app/SideCiSample6.php

```diff
@@ -0,0 +1,10 @@
+<?php
+class SideCiSample6 {
+public function hoge() { echo "hoge"; }
+public function fuga() {
```

Comment:
__[code_sniffer]__
* Line indented incorrectly; expected at least 4 spaces, found 0
* Opening brace should be on a new line